src/elements/core/i18n/i18n.ts

```diff
@@ -768,3 +768,17 @@ export const i18nDateMax = (max: string): Record<string, string> => ({
   fr: `La date ne doit pas être postérieure au ${max}.`,
   it: `La data non deve essere successiva al ${max}.`,
 });
+
+export const i18nChipDelete: Record<string, string> = {
+  de: 'TODO',
+  en: "Press the 'Delete' button to remove the chip",
+  fr: 'TODO',
```

Comment:
Please verify somehow
```suggestion
  fr: 'Appuyez sur la touche 'Del' pour supprimer cette puce',
```

Reply:
Did you forgot the german one?